edi_shipment.xml

```diff
@@ -0,0 +1,90 @@
+      </record>
+
+      <record model="ir.ui.view" id="edi_shipment_in_view_tree">
+          <field name="model">edi.shipment.in</field>
+          <field name="type">tree</field>
+          <field name="name">edi_shipment_tree</field>
+      </record>
+
+      <record model="ir.ui.view" id="edi_shipment_line_view_form">
+          <field name="model">edi.shipment.in.line</field>
+          <field name="type">form</field>
+          <field name="name">edi_shipment_line_form</field>
+      </record>
+
+      <record model="ir.ui.view" id="edi_shipment_line_view_tree">
+          <field name="model">edi.shipment.in.line</field>
+          <field name="type">tree</field>
+          <field name="name">edi_shipment_line_tree</field>
+      </record>
+
+      <record model="ir.action.act_window" id="act_edi_shipment_in_form">
+          <field name="name">Edi Shipment</field>
+          <field name="res_model">edi.shipment.in</field>
+      </record>
+
+      <record model="ir.action.act_window.view" id="act_edi_shipment_in_form_view1">
+          <field name="sequence" eval="10"/>
+          <field name="view" ref="edi_shipment_in_view_tree"/>
+          <field name="act_window" ref="act_edi_shipment_in_form"/>
+      </record>
+
+      <record model="ir.action.act_window.view" id="act_edi_shipment_in_form_view2">
+          <field name="sequence" eval="20"/>
+          <field name="view" ref="edi_shipment_in_view_form"/>
+          <field name="act_window" ref="act_edi_shipment_in_form"/>
+      </record>
+
+      <menuitem name="Edi Shipment" parent="stock.menu_stock"
+          id="menu_edi_shipment" sequence="40"/>
+
+      <menuitem parent="menu_edi_shipment"
+          action="act_edi_shipment_in_form"
+          id="menuitem_edi_shipment"
+          sequence="0" icon="tryton-list"/>
+
+
+      <record model="ir.ui.menu-res.group" id="menu_edi_shipment_in_group_account">
+          <field name="menu" ref="menu_edi_shipment"/>
+          <field name="group" ref="stock.group_stock_admin"/>
+      </record>
+
+      <record model="ir.model.button" id="create_shipment_button">
+          <field name="name">create_shipment</field>
+          <field name="model" search="[('model', '=', 'edi.shipment.in')]"/>
+      </record>
+
+      <record model="ir.model.button-res.group"
+          id="create_invoice_button_group_account">
+          <field name="button" ref="create_shipment_button"/>
+          <field name="group" ref="stock.group_stock_admin"/>
+      </record>
+
+        <record model="ir.model.button" id="search_related_button">
+          <field name="name">search_references</field>
+          <field name="model" search="[('model', '=', 'edi.shipment.in')]"/>
+      </record>
+      <record model="ir.model.button-res.group"
+          id="search_related_button_group_account">
+          <field name="button" ref="search_related_button"/>
+          <field name="group" ref="stock.group_stock_admin"/>
+      </record>
+
```

Comment:
No se si fora bo afegir una regla d'accés per empresa a 'edi.shipment.in' ja que té el camp "company".

suposo que està pensat a nivell de importació de edis en dos empreses, definir dos crons, cada un a una empresa, i a stock.configuration decidir el path dels fitxer per cada empresa.

Veure punt que comento el Multivalue.